sdk/cosmos/azure-cosmos-spark_3_2-12/docs/quick-start.md

```diff
@@ -26,19 +26,19 @@ You can use any other Spark 3.1.1 spark offering as well, also you should be abl
 SLF4J is only needed if you plan to use logging, please also download an SLF4J binding which will link the SLF4J API with the logging implementation of your choice. See the [SLF4J user manual](https://www.slf4j.org/manual.html) for more information.
 
 For Spark 3.1:
-- Install Cosmos DB Spark Connector, in your spark Cluster [com.azure.cosmos.spark:azure-cosmos-spark_3-1_2-12:4.30.0](https://search.maven.org/artifact/com.azure.cosmos.spark/azure-cosmos-spark_3-1_2-12/4.30.0/jar)
+- Install Cosmos DB Spark Connector, in your spark Cluster [com.azure.cosmos.spark:azure-cosmos-spark_3-1_2-12:4.32.0](https://search.maven.org/artifact/com.azure.cosmos.spark/azure-cosmos-spark_3-1_2-12/4.32.0/jar)
```

Comment:
sorry I missed this change, but shouldn't this be 4.31.0?

Reply:
Yeah - fixed